extra/supervisord/celeryd.conf

```diff
@@ -24,10 +24,8 @@ startsecs=10
 ; Increase this if you have very long running tasks.
 stopwaitsecs = 600
 
-; When resorting to send SIGKILL to the program to terminate it
-; send SIGKILL to its whole process group instead,
-; taking care of its children as well.
-killasgroup=true
+; Causes supervisor to send the stop signal to the whole process group.
```

Comment:
Thanks for looking into this. Do you mind updating this comment to use `send the termination` or `send the SIGTERM`?